.circleci/scripts/binary_checkout.sh

```diff
@@ -63,6 +63,8 @@ popd
 # Clone the Builder master repo
 retry git clone -q https://github.com/pytorch/builder.git "$BUILDER_ROOT"
 pushd "$BUILDER_ROOT"
+git fetch
+git checkout lts/release/1.8
```

Comment:
Nit: one can specify branch to clone from directly as part of `git clone` command
```suggestion
retry git clone -q https://github.com/pytorch/builder.git -b lts/release/1.8 "$BUILDER_ROOT"
pushd "$BUILDER_ROOT"
```